src/cic/model.clj

```diff
@@ -54,8 +54,9 @@
                               c (get model-coefs (str "quarter:admission_age" age) 0.0)
                               y (try (m/exp (+ intercept a (* b day) (* c day)))
                                      (catch Exception e
-                                       (print intercept a age model-coefs)
-                                       (throw e)))
+                                       (throw (ex-info "Model params missing: "
```

Comment:
`"Model params missing."` rather than with a colon as it won't show up that way in the debugger. 

Is there anything else that should go into that map with the exception so that we can figure out what went wrong and how to fix it?

Reply:
Based on my message it would make sense to put all the params in the map, but in this case we observed that one or more of the params were empty, which helped lead us to the fact that the file we were reading in was empty (although it did exist)

Reply:
If the params are really big, then we probably want to put them in an atom we can insepect from the repl if this throws an exception, and the message should say which atom to look at. This is what we do in witan.send

```clojure
(try
    (cond
      (= state c/non-send)
      population-by-state
      (or (<= year sc/min-academic-year)
          (> year sc/max-academic-year))
      [model
       (cond-> transitions
         (pos? population)
         (update [calendar-year year state c/non-send] m/some+ population))]
      :else
      (apply-leavers-movers-for-cohort-unsafe population-by-state cohort params calendar-year
                                              valid-transitions make-setting-invalid))
    (catch Exception e
      (reset! exception-info {:population-by-state population-by-state
                              :cohort cohort
                              :params params
                              :calendar-year calendar-year
                              :valid-transitions valid-transitions
                              :make-setting-invalid make-setting-invalid
                              :exception e})
      (throw (ex-info "Could not calculate leavers and movers for cohort. See the atom witan.send.model.run/@exception-info for details."
                      {:cohort cohort}
                      e))))
```

Reply:
in run.clj

Reply:
then you can use the data in the exception to poke at the atom and see if things are set up as you expect.

Reply:
They're not. Each param is just a number